src/bttrx_display.h

```diff
@@ -0,0 +1,43 @@
+/*
+This file is part of bt-trx
+
+bt-trx is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+bt-trx is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright (C) 2019 Christian Obersteiner (DL1COM), Andreas Müller (DC1MIL),
+              2020 Florian Wolters (DF2ET)
+Contact: bt-trx.com, mail@bt-trx.com
+*/
+
+#pragma once
+
+#ifdef ARDUINO
+
+#define display_width  128
+#define display_height  64
+
+#include <string>
+using namespace std;
+
+extern const char *GIT_REVISION;
```

Comment:
Please include GIT_REVISION in bttrx_display.cpp as it is not used in the header, only in the cpp file